### PR TITLE
merge with SpaceK33z

### PR DESCRIPTION
**Pull request by Duangboudda**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by Duangboudda** [comment]
Merge original iohook with SpacK33z fork

**Comment by lacymorrow** [comment]
Hello!
Could you tell us a bit more about this PR?

File: install.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ function install(runtime, abi, platform, arch, cb) {
 
   console.log('Downloading prebuild for platform:', currentPlatform);
   let downloadUrl =
-    'https://github.com/wilix-team/iohook/releases/download/v' +
+    'https://github.com/SpaceK33z/iohook/releases/download/v' +
```
Comment (lacymorrow) [inline install.js:34]: Should we be referencing this?

File: package.json [hunk 2]
```diff
@@ -1,16 +1,16 @@
 {
-  "name": "iohook",
-  "version": "0.9.3",
+  "name": "@spacek33z/iohook",
```
Comment (lacymorrow) [inline package.json:2]: update

File: package.json [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+  "version": "0.12.0",
```
Comment (lacymorrow) [inline package.json:3]: update

File: package.json [hunk 4]
```diff
@@ -80,104 +80,34 @@
...
+      "17.0.0",
+      "101"
     ],
     [
-      "node",
-      "16.0.0",
-      "93"
+      "electron",
+      "18.0.0",
+      "103"
```
Comment (lacymorrow) [inline package.json:110]: why are we dropping support for old versions here?

**Review by lacymorrow — CHANGES_REQUESTED** [review]

**Comment by Duangboudda** [comment]
Hi,

Sorry,

I made PR by mistake; I meant to merge into repo that I forked, so that I can see the different

Thanks
Tong
________________________________

**Comment by Duangboudda** [comment]
I was working to get iohook to work with electron 28, and we did get it work. And we want to know that if the codes are safe to use, mostly just scanning code for security reasons.

**Comment by Duangboudda** [comment]
Yeah, I will close it, and sorry for the mistake. it is great library and we use it in our organization. Currently, we update our electron version to 28, so we found some forked repo to use, but we want to see new committed codes for security purposes.

**Pull request closed without merging**